### PR TITLE
fix(curriculum): standardize quotations

### LINKED ISSUE #56081: Standardize Quotation Marks in Attribute Values
### Describe the Issue

Without quotation marks on the type attribute value, all tests pass successfully. However, it is the opposite when the value is in quotation marks. 

Having quotation marks on attribute values is standard and considered best practice. 

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-form-validation-by-building-a-calorie-counter/step-69

### Your code

```

function calculateCalories(e) {
  e.preventDefault();
  isError = false;
  const breakfastNumberInputs = document.querySelectorAll("#breakfast input[type='number']");
}

```

### Expected behavior

With or without quotation marks, the test should pass.

### Screenshots

Screenshot 2024-09-12 165412

### System

- Browser: [e.g. Chrome, Safari]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56081

### CODE REVIEW

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
changes are needed also for the other selectors that have the same issue, and it should be added to the seed code as having the quotes is the preferred way

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9c0d0857f0a10a57af936.md [hunk 1]
```diff
@@ -56,13 +56,13 @@ assert.match(calculateCalories.toString(), /exerciseNumberInputs\s*=/);
 You should call `document.querySelectorAll()` with `#exercise input[type=number]`.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9c0d0857f0a10a57af936.md:56]: Please continue finding all the places in which a change is needed

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9c0d0857f0a10a57af936.md [hunk 2]
```diff
@@ -38,13 +38,13 @@ assert.match(calculateCalories.toString(), /snacksNumberInputs\s*=/);
...
 ```js
-assert.match(calculateCalories.toString(), /document\.querySelectorAll\(\s*('|")#snacks input\[\s*type\s*=\s*number\s*\]\s*\1\s*\)/);
+assert.match(calculateCalories.toString(), /document\.querySelectorAll\(\s*('|"|`)#snacks input\[\s*type\s*=\s*('|")?number('|")?\s*\]\s*\1\s*\)/);
 ```
 
 You should assign the result of your `document.querySelectorAll()` call to `snacksNumberInputs`.
 
 ```js
-assert.match(calculateCalories.toString(), /snacksNumberInputs\s*=\s*document\.querySelectorAll\(\s*('|")#snacks input\[\s*type\s*=\s*number\s*\]\s*\1\s*\)/);
+assert.match(calculateCalories.toString(), /snacksNumberInputs\s*=\s*document\.querySelectorAll\(\s*('|"|`)#snacks input\[\s*type\s*=\s*('|")?number('|")?\s*\]\s*\1\s*\)/);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9c0d0857f0a10a57af936.md:47]:
```suggestion
assert.match(calculateCalories.toString(), /snacksNumberInputs\s*=\s*document\.querySelectorAll\(\s*('|"|`)#snacks input\[\s*type\s*=\s*('|")?number\2?\s*\]\s*\1\s*\)/);
```
use a capture group to match the same type of quote, this needs to be changed in all the tests

**Comment by gagan-bhullar-tech** [comment]
@ilenia-magoni thanks. fixed all instances.

**Review by majestic-owl448 — APPROVED** [review]
looks good, thank you for the PR!

**Review by Sembauke — APPROVED** [review]

**Pull request merged**